docs/hermes.md

```diff
@@ -89,12 +89,14 @@ If you've recently created a new app from scratch, you should see if Hermes is e
 
 ![Where to find JS engine status in AwesomeProject](/docs/assets/HermesApp.jpg)
 
-A `HermesInternal` global variable will be available in JavaScript that can be used to verify that Hermes is in use:
+A `HermesInternal` global variable will be available in JavaScript that can be used to verify that Hermes is in use*:
 
 ```jsx
 const isHermes = () => !!global.HermesInternal;
 ```
 
+*If you are using a non-standard way of loading the JS bundle, it is possible that the `HermesInternal` variable is available but you aren't using the hermes generated bytecode. Confirm that you are using the `.hbc` file and also benchmark the before/after as detailed below.
```

Comment:
I think it's a bit confusing to use **but**, since `HermesInternal` only means you are using Hermes engine, bytecode usage is another thing on top of that and it doesn't contradict whether Hermes is used or not.

imo this not probably makes more sense below the line: `This will compile JavaScript to bytecode during build time which will improve your app's startup speed on device.` as it's related.

Reply:
Instead using asterisk (is not the common pattern in docs), pleas wrap this information with blockquote (via `>`). 

This will display the message on the website in the form of note. 

With the wording correction, and with the display tweak in, this PR should be ready to merge. 🙂